### PR TITLE
Fix payable example.

### PR DESCRIPTION
**Pull request by chriseth**

### CODE REVIEW

**Review by hrkrshnn — DISMISSED** [review]

File: docs/contracts/functions.rst [hunk 1]
```diff
@@ -438,7 +438,10 @@ operations as long as there is enough gas passed on to it.
             // results in test.x becoming == 1 and test.y becoming 1.
 
             // If someone sends Ether to that contract, the receive function in TestPayable will be called.
-            require(payable(test).send(2 ether));
+            // Since that function writes to storage, it takes more gas than is available with a
+            // simple ``send`` or ``transfer``. Because of that, we have to use a low-level call.
+            (success,) = address(test).call{value: 1}("");
```
Comment (leonardoalt) [inline docs/contracts/functions.rst]: Isn't `value: 1` even less than what `send` and `transfer` forward?

Comment (chriseth) [inline docs/contracts/functions.rst]:
```suggestion
            (success,) = address(test).call{value: 2 ether}("");
```

Comment (hrkrshnn) [reply docs/contracts/functions.rst]: What do you mean?

Comment (leonardoalt) [reply docs/contracts/functions.rst]: value: 1 means 1 wei is being sent, which is usually less than 1 gas, whereas send and transfer forward 2300 gas to the called receive/fallback function

**Review by hrkrshnn — DISMISSED** [review]

File: docs/contracts/functions.rst (hunk 1 above)
Comment (hrkrshnn) [reply docs/contracts/functions.rst]: Ok. Sending 2 ether is a more practical example.

**Pull request merged**